### PR TITLE
handle multi-line and comma-separated @see annotations

### PR DESCRIPTION
**Pull request by kedashoe**
see https://github.com/ramda/ramda/pull/1256#issuecomment-119698147

### CODE REVIEW

File: lib/doc/jsdoc-template/publish.js [hunk 1]
```diff
@@ -36,6 +36,24 @@ var prettifyCode = R.pipe(
...
+      R.map(
+        R.pipe(
+          R.trim,
+          unprefix
+        )
+      )
+    )
+  ),
+  R.flatten
+);
```
Comment (davidchambers) [inline lib/doc/jsdoc-template/publish.js]: Could you provide some example input–output pairs for this function? I'm pretty sure we can simplify this a bit, perhaps by using `R.chain`.

Comment (kedashoe) [inline lib/doc/jsdoc-template/publish.js]: input: `['R.one', 'R.two, R.three']`
output: `['one', 'two', 'three']`

also I'm in ramda's gitter, maybe easier if you jump in there?

**Comment by davidchambers** [comment]
:deciduous_tree:

**Comment by buzzdecafe** [comment]
:cow2: 

(`seemplify`?)

**Comment by CrossEye** [comment]
:herb:

I'd boo you @buzzdecafe, but I"m afraid it'd turn into a moo.

**Comment by buzzdecafe** [comment]
&mu;

**Pull request merged**